### PR TITLE
ENG-11729

### PR DESCRIPTION
**Pull request by akhanzode**
Remove old export syntax and DDL

### CODE REVIEW

File: src/frontend/org/voltcore/network/InputHandler.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
      * Handle the incoming message produced by retrieve next message
      *
      * @param message
+     * @param c
      * @param channelFacade
```
Comment (xinjiacs) [inline src/frontend/org/voltcore/network/InputHandler.java]: @akhanzode Remove the "channelFacade"?

**Comment by xinjiacs** [comment]
Do you test that "EXPORT TABLE" syntax  can not be recognized?

**Comment by akhanzode** [comment]
No I dont its as good as garbage ddl syntax. I can add a test easily if you like.

File: src/frontend/org/voltdb/compiler/VoltCompiler.java [hunk 2]
```diff
@@ -945,7 +945,13 @@ private void compileDatabase(
         // Actually parse and handle all the DDL
         // DDLCompiler also provides partition descriptors for DDL PARTITION
         // and REPLICATE statements.
-        final DDLCompiler ddlcompiler = new DDLCompiler(this, hsql, voltDdlTracker, m_classLoader);
+        final DDLCompiler ddlcompiler;
+        try {
+            ddlcompiler = new DDLCompiler(this, hsql, voltDdlTracker, m_classLoader);
+        } catch (Throwable t) {
```
Comment (akhanzode) [inline src/frontend/org/voltdb/compiler/VoltCompiler.java]: good catch leftover for debug.

**Comment by zeemanhe** [comment]
There are tons of comments or debug statements in the codes using "export table" instead of "stream". It is not worth updating them.  

Should the following be updated with "stream"?
line 213 in  ReportMaker.generateSchemaRow() 
line 282 in CatalogBuilder.setTableAsExportOnly()

**Pull request merged**